content/en/security/cloud_workload_security/security_profiles.md

```diff
@@ -0,0 +1,78 @@
+
+## How Workload Security Profiles are generated
+
+Workload Security Profiles are generated when [activity snapshots](#activity-snapshots) from containers sharing a common image name and image tag (version) are merged together to form a security profile. A different security profile is generated for each version of a container image. As a result, one profile is generated per `{ image-name, image-version}` tag combination.
+
+Each security profile acts as a [behavior model](#behavior-learning-model) for a given workload image to identify known, acceptable behavior that is expected to be consistent across all containers versus previously unseen, anomalous behavior that may be unique to a single container or small number of containers.
+
+### Activity snapshots
+
+Activity snapshots are the building blocks of Workload Security Profiles. They provide a reasonably accurate representation of expected workload behavior. Each snapshot captures kernel-level activity on a container, including process, network, and file access details. This information is collected by the Agent in 30 minute intervals and sent to the Datadog backend.
+
+When you configure the Agent and enable activity dumps, it automatically starts capturing snapshots for any running cgroups, including those already running when you start the Agent. It also captures snapshots for any new containers in your cloud native environments that use the Linux kernel control groups or cgroups.
+
+Each Agent can profile multiple containers on the host system simultaneously. To minimize performance overhead, no more than five containers are profiled at any given time.
+
+### Behavior learning model
+
+A Workload Security Profile is a behavior model for a specific workload image. Capturing and comparing activity snapshots from individual containers enables each security profile to provide a model of the aggregate workload behavior of similar containers. This representation results in a more precise behavior model for a specific workload as more containers are profiled. Since containerized workloads are expected to be immutable, there should be few variations between workloads that share common image name and image version tags.
+
+A visual representation of the behavior model, including process nodes and file and network activity relationships is shown on the [security profile details page](#explore-security-profiles). The profile visualization displays how consistent or common these relationships are across the various containers that have been profiled.
+
+#### Model status
+
+While initial snapshots are being captured and merged into the Workload Security Profile, the profile will be shown with a status of **learning**. After a threshold has been reached in terms of the number of activity snapshots merged or amount of time elapsed since the first snapshot, the security profile status will transition to a **stable** state.
```

Comment:
What's the threshold for stable state, or is that determined by the user?

Reply:
I actually have the same question out to Product right now. I agree, we definitely need to be more specific here.